src/cexecutor/LibaryLoader.cpp

```diff
@@ -14,6 +17,7 @@ using yacx::loglevel, yacx::detail::dynop, yacx::detail::opfn;
 
 void yacx::detail::load_op(struct dynop *dest, std::string filename, std::string opSymbolName) {
     //open libary
+    #ifndef _MSC_VER
```

Comment:
not the most elegant solution, but linking libraries is vastly different under windows. we could use something like this: https://github.com/alainfrisch/flexdll

not sure how to properly say "hey these functions don't work under windows" so for now they just do nothing

Reply:
@LukasSiefke this is so sad

Reply:
sollte wahrscheinlich halt ne *not implemented exception* werfen oder so